### PR TITLE
Project module support for skip_delete flag on google_project resource

### PR DESCRIPTION
**Pull request by christopher-avila-ditoweb**
We have a use case where we want to create short-lived projects with this module but did not want the project to be removed when cleaning up resources. Since the google_project resource had an optional skip_delete flag, it felt to me that maintaining a fork was overkill for such a small change and the cleanest solution was to add it to the repository with a default that would allow the modules current behavior to remain the same while optionally adding the desired functionality.

### CODE REVIEW

**Comment by christopher-avila-ditoweb** [comment]
@googlebot I signed it!

File: modules/project/variables.tf [hunk 1]
```diff
@@ -162,6 +162,12 @@ variable "project_create" {
   default     = true
 }
 
+variable "project_skip_delete" {
```
Comment (juliocc) [inline modules/project/variables.tf:165]: We're already in the project module, so the "project" prefix seems redundant. Can we call it just `skip_delete`?

**Review by juliocc — CHANGES_REQUESTED** [review]
Nice! Left one small comment, otherwise LGTM

**Review by juliocc — APPROVED** [review]

**Comment by juliocc** [comment]
Thanks for your contribution!

**Pull request merged**